state/state.go

```diff
@@ -801,20 +801,7 @@ func (st *State) FindEntity(tag names.Tag) (Entity, error) {
 		// Return an invalid entity error if the requested model is not
 		// the current one.
 		if id != model.UUID() {
-			if utils.IsValidUUIDString(id) {
-				return nil, errors.NotFoundf("model %q", id)
-			}
```

Comment:
This returned not found, we should do the same.

Reply:
Done